orchagent/portsorch.cpp

```diff
@@ -2943,6 +2949,9 @@ bool PortsOrch::removeLag(Port lag)
         SWSS_LOG_ERROR("Failed to remove LAG %s, it is still in VLAN", lag.m_alias.c_str());
         return false;
     }
+    /* notify PORTCHANGE before remove port*/
```

Comment:
Same question, if we notify other classes before we remove_lag on HW, what if the remove_lag failed?

Reply:
Same as the previous reply